brainglobe_stitch/image_mosaic.py

```diff
@@ -340,3 +346,366 @@ def read_big_stitcher_transforms(self) -> None:
         for tile in self.tiles:
             stitched_position = stitched_translations[tile.id]
             tile.position = stitched_position
+
+    def fuse(
+        self,
+        output_file_name: str = "fused.zarr",
+        downscale_factors: Tuple[int, int, int] = (1, 2, 2),
+    ) -> None:
+        """
+        Fuse the tiles into a single image and save it to the output file.
+
+        Parameters
+        ----------
+        output_file_name: str, default: "fused.zarr"
+            The name of the output file, suffix dictates the output file type.
+        downscale_factors: Tuple[int, int, int], default: (1, 2, 2)
+            The factors to downscale the image by in the z, y, x dimensions.
```

Comment:
There wasn't a specific reason, initially I had the downscale factors be (2, 2, 2) in z, y, x however, I once got a dataset which where the resolution was wildly different between z and y/x (50, 5, 5).  This caused issues when downsampling by (2, 2, 2) as BigStitcher uses the downsampled version to stitch the tiles together and it did a terrible job when the z resolution was so low. Because of that I decided to set the default as (1, 2, 2) as the z resolution is typically lower (and the size was also typically lower) it didn't incur too big of a penalty in terms of data size.

Reply:
Okay nice that makes sense, I wonder if it is worth a brief explanation of why this default was chosen in the docstring. But maybe this is TMI! I'll leave it up to you